### PR TITLE
[APPWIZ] Romanian translation update

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-romanian 

Also, if you have translations of multiple modules, it might be better to have all in one Pull Request, so it could be reviewed in one place.

**Comment by AndreiMiloiu** [comment]
Yes, but I don't know how to add more files in a pull request.

**Comment by AndreiMiloiu** [comment]
I'm still not able to add two or more files in a single patch and in a single Pull Request.

**Comment by AndreiMiloiu** [comment]
Finally, I got to know how to add more files in a single Pull Request.

File: dll/cpl/appwiz/lang/ro-RO.rc [hunk 1]
```diff
@@ -88,8 +89,8 @@ BEGIN
     IDS_DWL_FAILED "Descărcarea pachetului Gecko a eșuat. Asigurați-vă că aveți conexiune la internet pentru a putea descărca. Asistentul va continua fără a instala pachetul Gecko."
     IDS_CANTMAKEINETSHORTCUT "Eșec în crearea scurtăturii la Internet."
     IDS_CANTMAKESHORTCUT "Eșec în crearea scurtăturii."
-    IDS_NO_MEMORY "No memory could be allocated!"
-    IDS_NO_DIRECTORY "No directory given!"
-    IDS_INVALID_PATH "The given path is invalid!"
-    IDS_INVALID_NAME "The shortcut name you entered either contained characters that are invalid for file names or was too long."
+    IDS_NO_MEMORY "Nu s-a putut aloca memorie!"
```
Comment (reactos-contributor) [inline dll/cpl/appwiz/lang/ro-RO.rc:92]: Consider using the passive voice (rom. diateza pasivă) for such cases (see the 15th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)).
I.e. „Nu a putut fi alocată memorie!”

File: dll/cpl/appwiz/lang/ro-RO.rc [hunk 2]
```diff
@@ -88,8 +89,8 @@ BEGIN
...
+    IDS_NO_DIRECTORY "Nu a fost dat niciun director!"
+    IDS_INVALID_PATH "Calea dată este invalidă!"
```
Comment (reactos-contributor) [inline dll/cpl/appwiz/lang/ro-RO.rc:94]: Although personally I have no issue with „invalid” in Romanian being used with the same meaning as in English, I must mention that other translators strongly recommend using „nevalid” instead (and leave „invalid” term only for referring to infirm persons).

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

**Review by reactos-contributor — APPROVED** [review]

File: dll/cpl/appwiz/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply dll/cpl/appwiz/lang/ro-RO.rc]: Everything is fixed now.

**Comment by AndreiMiloiu** [comment]
As there are no issues anymore, I think this PR can be merged.

File: dll/cpl/appwiz/lang/ro-RO.rc [hunk 3]
```diff
@@ -1,7 +1,10 @@
 /*
- * FILE:             dll/cpl/appwiz/lang/ro-RO.rc
- *                   ReactOS Project (http://www.reactos.org)
- * TRANSLATOR:       Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS Application Wizard
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2009 Petru Dimitriu <petrimetri@gmail.com>
+ *              Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2022 Andrei Miloiu <miloiuandrei@gmail.com>
```
Comment (binarymaster) [inline dll/cpl/appwiz/lang/ro-RO.rc:7]:
```suggestion
 * PROJECT:     ReactOS Software Control Panel
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Romanian resource file
 * TRANSLATORS: Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
 *              Copyright 2022 Andrei Miloiu <miloiuandrei@gmail.com>
```

**Review by binarymaster — APPROVED** [review]

**Pull request merged**